src/test/java/test/dependent/github1380/GitHub1380Sample4.java

```diff
@@ -0,0 +1,21 @@
+package test.dependent.github1380;
```

Comment:
@juherr - I was asking about the nature of the test and not the behavior. Is that test semantically correct ? Why would a person want to mix up `dependsOnMethods` and `priority` in a circular fashion (Visibly its like going around in circles). So I wanted to know, if that test case by itself is semantically correct.